src/Widget/Conversation/Header/index.tsx

```diff
@@ -5,6 +5,7 @@ const close = require('../../../../assets/clear-button.svg').default as string;
 
 import './style.scss';
 import { GlobalState } from '@types';
+import { formatPhoneNumber } from '../../../utils/common';
```

Comment:
For using absolute path in this project you should begin the string from `src`:
`import { formatPhoneNumber } from 'src/utils/common';`